common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/config/PreDistributionTrlValueMappingProvider.java

```diff
@@ -0,0 +1,42 @@
+package app.coronawarn.server.common.persistence.domain.config;
+
+import app.coronawarn.server.common.persistence.utils.YamlPropertySourceFactory;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@ConfigurationProperties()
+@PropertySource(value = "classpath:trl-value-mapping-1.15.yaml",
+    factory = YamlPropertySourceFactory.class)
+public class PreDistributionTrlValueMappingProvider {
```

Comment:
`PreDistributionTrlValueMappingProvider` name is a bit misleading, when we use it during the submission process, right?

Reply:
I agree. But somehow that is the context where the mapping is used for. My idea was to put this explicit naming to force us to think if there is a design issue we are missing here. Together with Max we did discuss other aproaches where the configuration can not be shared, but to be honest, the most optimal way is to filter out checkins during submission based on this "distribution-related" configuration. 
We could change the naming or revisit this at a later point in time.